app/Global.scala

```diff
@@ -29,7 +32,7 @@ object Global extends GlobalSettings with LazyLogging{
     startActors(conf.underlying, app)
 
     if (conf.getBoolean("application.insertInitialData") getOrElse false) {
-      InitialData.insert(conf)
+      InitialData.insert
```

Comment:
Somebody should check the result of this. Otherwise all the `Fox`ness below is useless

Reply:
Not sure how to do that, since `onStart` itself returns unit? I’m open to suggestions.

I disagree though, that the Foxness is useless without that. I added it to ensure the order of the insert steps (user is needed before project, for example). 

As I understand it, your suggestion would additionally ensure that all of this is done before the first page load, but that is not as critical since in the worst case a browser refresh would be necessary.

Reply:
I was rather worried about ignoring error that might occur during initialization than about order. But I also don't really know what should happen if this fails. Throw an exception? exit?

Reply:
I see your point but I think silent failure here is no big problem, since this is only used for local development and you will see pretty quickly if scmboy does not exist. I’ll add printing an error log if that is fine by you